lib/app.js

```diff
@@ -1,53 +1,44 @@
 (function (factory) {
   if (typeof define === 'function' && define['amd']) {
-    define(['knockout', './calculator', './todo-list'], factory);
+    define(['knockout'], factory);
   } else if (typeof require === 'function' && typeof exports === 'object' && typeof module === 'object') {
-    factory(require('knockout'), require('./calculator'), require('./todo-list'));
+    factory(require('knockout'));
   } else {
-    factory(ko, Calculator, ToDoList);
+    factory(ko);
   }
-} (function (ko, Calculator, ToDoList) {
+} (function (ko) {
```

Comment:
We no longer have direct dependencies on Calculator or ToDoList so I removed them from here. Knockout loads these whenever we use the components.